sdk/txbuild/formation.go

```diff
@@ -31,6 +31,10 @@ func Formation(p FormationParams) (*txnbuild.Transaction, error) {
 		HighThreshold:   txnbuild.NewThreshold(2),
 		Signer:          &txnbuild.Signer{Address: p.ResponderSigner.Address(), Weight: 1},
```

Comment:
in [the SEP Reserves Section](https://github.com/stellar/experimental-payment-channels/blob/main/specifications/sep-payment-channel-mechanism.md#reserves) we say `I` is responsible for sponosring the reserves of `I` only. But here `I` sponsoring `R` for `EI`. I think this makes sense, instead of creating another Sponsor operation sandwich for `ER` sponsored by `I`. Since it evens out anyways and the escrow accounts eventually will be destroyed

Maybe a better way of saying it is "I is responsible for sponsoring the reserves of EI"?

Reply:
Oh hmm, this is a bug in this code.

Reply:
It evens out unless I and R have different numbers of signers. We shouldn't need new sponsor operations, we just need to reorder the inner operations into the other blocks I think?

Reply:
true I didn't think about if I and R have more than 1 signer, in that case the wording makes sense as is then

Reply:
gonna merge since the example code change can be committed separately